po/de.po

```diff
@@ -3954,58 +3937,51 @@ msgid ""
 msgstr ""
 
 #: src/exercises/day-1/for-loops.md:80
-#, fuzzy
 msgid "## Bonus Question"
 msgstr "## Bonus-Frage"
 
 #: src/exercises/day-1/for-loops.md:82
-#, fuzzy
 msgid ""
 "Could you use `&[i32]` slices instead of hard-coded 3 × 3 matrices for your\n"
 "argument and return types? Something like `&[&[i32]]` for a two-dimensional\n"
 "slice-of-slices. Why or why not?"
 msgstr ""
-"Könnten Sie `&[i32]`-Slices anstelle von fest codierten 3 × 3-Matrizen für "
-"Ihre verwenden\n"
-"Argument- und Rückgabetypen? Etwas wie `&[&[i32]]` für ein "
-"zweidimensionales\n"
-"Scheibe-von-Scheiben. Warum oder warum nicht?"
+"Könntest Du `&[i32]`-Slices anstelle von fest codierten 3 × 3-Matrizen als\n"
+"Argument- und Rückgabetypen verwenden? Oder etwas wie `&[&[i32]]` für ein "
+"zweidimensionales Slice? \n"
```

Comment:
The German translation of the Rust Programming Language book uses "Anteilstyp" as translation for Rust; https://rust-lang-de.github.io/rustbook-de/ch04-03-slices.html

Reply:
The idea was to start with the german translations and then slowly change to the english once as one progresses further, because these are the names you will inevitably use. However, I see that this is confusing and harder to understand because you suddenly have two names for the same thing. I will adhere stricktly to the german names in my following PRs.